extensions/pkg/webhook/handler_shootclient.go

```diff
@@ -162,11 +175,11 @@ func (h *handlerShootClient) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	req := admission.Request{}
-	ar := admissionv1beta1.AdmissionReview{
+	ar := admissionv1.AdmissionReview{
```

Comment:
Arg, I missed to change that part again similar to the seed-admission-controller. It should rather use the upstream c-r decoding logic instead of the custom implementation.

Reply:
Addressed in the latest commit.

Reply:
Just to be sure: We don't have a problem here, right? It would still work with v1.10 clusters because the `admissionv1` objects are only used internally in the webhook code?

Reply:
Yes, we use the c-r handlers now which take over decoding and caring about the right version of the `admission` API.